docs/design/solutions/hub-and-spoke-tpe/flow_create_install.txt

```diff
@@ -0,0 +1,55 @@
+@startuml
+title FLOW create
+
+participant Client
+participant NorthBound as NB
+control kilda.northbound
+participant "FLOW SplitterBolt" as SplitterBolt
+participant "FLOW CrudBolt" as CrudBolt
+participant "FLOW ErrorBolt" as ErrorBolt
+participant "FLOW NorthboundReplyBolt" as NorthboundReplyBolt
+participant "FLOW FlowCommandBolt" as FlowCommandBolt
+participant FlowService
+participant DAO
+participant "FLOW TransactionBolt" as TransactionBolt
+control kilda.flow
+participant "FLOW SpeakerBolt" as SpeakerBolt
+
+Client -> NB: /flows PUT
+NB -> kilda.flow: CommandMessage\nFlowCreateRequest
+kilda.flow -> SplitterBolt: CommandMessage\nFlowCreateRequest
+SplitterBolt -> CrudBolt: CommandMessage\nFlowCreateRequest
+note right of CrudBolt: validate the flow (using FlowValidator)
+note right of CrudBolt: new path (using PathComputer.getPath)
+note right of CrudBolt: build flow pair (state=ALLOCATED)
+note right of CrudBolt: save flow (FlowCache.createFlow)
+note right of CrudBolt: allocate cookie for flow
+note right of CrudBolt: allocate transit vlan (forward)
+note right of CrudBolt: allocate transit vlan (reverse)
+note right of CrudBolt: new FlowInfoData
+CrudBolt -> FlowCommandBolt: InfoMessage.FlowInfoData
+CrudBolt -> NorthboundReplyBolt:  FlowInfoData
+alt error
+	CrudBolt -> ErrorBolt: ErrorMessage
+	ErrorBolt -> NorthboundReplyBolt: ErrorMessage
+end
+NorthboundReplyBolt -> kilda.northbound: InfoMessage or\n ErrorMessage
+
+loop in forward, reveres
```

Comment:
Are you going to ignore error from DB(DAO) layer?

PS Don't see where we storing flow in DB.

Reply:
see loop above